test/e2e/beta/metamask-beta-ui.spec.js

```diff
@@ -271,17 +284,17 @@ describe('MetaMask', function () {
       await driver.wait(until.stalenessOf(accountModal))
       await delay(regularDelayMs)
     })
-    it('show account details dropdown menu', async () => {
+    // it('show account details dropdown menu', async () => {
```

Comment:
Should we just remove these lines or add a comment on why they've been commented out?

Reply:
They were moved to a separate file in my PR that created e2e tests for the extension view. I just rebased, so they are gone now.